plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -1693,14 +1693,14 @@ private static Document getDocumentWithDifferentDbRefFieldOrder(Object objectId)
     {
         return new Document()
                 .append("_id", new ObjectId("5126bbf64aed4daf9e2ab771"))
-                .append("creator", new Document().append("collectionName", "doc_creators").append("id", objectId).append("databaseName", "doc_test"));
+                .append("creator", new Document().append("$ref", "doc_creators").append("$id", objectId).append("$db", "doc_test"));
```

Comment:
Since we're constructing the BSON document by hand here I changed these to actually correspond to correct MongoDB names for collectionName, databaseName etc. Otherwise these documents are in no way related to DBRefs.